commit-graph.c

```diff
@@ -713,6 +713,7 @@ static void close_commit_graph_one(struct commit_graph *g)
 	if (!g)
 		return;
 
+	clear_commit_graph_data_slab(&commit_graph_data_slab);
```

Comment:
@derrickstolee could I ask you to look over this change, to see whether it is sound?

Reply:
Makes sense. If we are closing the commit-graph, then the graph might change when we reopen it, leading to different `graph_pos` values.